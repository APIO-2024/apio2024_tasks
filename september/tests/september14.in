txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 0 2 3 2 2
5 6 4 1 3 2
5 6 4 1 3 2
6 5 4 1 3 2
5 6 4 1 3 2
5 6 4 1 3 2
10 5
0 1 0 2 3 2 2 5 8
9 4 8 7 5 3 2 6 1
9 4 5 3 8 7 2 6 1
9 4 3 7 5 8 6 2 1
9 7 3 5 8 4 2 6 1
9 8 7 3 4 5 2 6 1
7 5
0 0 1 1 1 3
6 5 3 2 1 4
6 5 2 3 1 4
6 5 3 2 1 4
5 6 4 2 3 1
5 6 1 2 3 4
6 5
0 0 0 3 4
5 1 4 2 3
5 1 4 3 2
5 1 4 2 3
5 1 4 2 3
5 1 4 2 3
