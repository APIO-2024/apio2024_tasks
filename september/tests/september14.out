7ckgnn4wyi495puj3ibqf81dqvapyv6b
5
4
2
4
